src/game/SpellScriptsTBC.cpp

```diff
@@ -0,0 +1,75 @@
+#include "SpellScriptFactory.h"
+
+extern ScriptedSpellFactory* spellFactories[MAX_TBC_SPELL_ID];
+
+enum
+{
+    SPELL_MENTAL_INTERFERENCE = 38915
+};
+
+/*Spell 38915*/
+struct MentalInterference : public Spell
+{
+    MentalInterference(Unit* caster, SpellEntry const* info, bool triggered, ObjectGuid originalCasterGUID = ObjectGuid(), SpellEntry const* triggeredBy = nullptr)
+        : Spell(caster, info, triggered, originalCasterGUID, triggeredBy)
```

Comment:
http://en.cppreference.com/w/cpp/language/using_declaration

For making this a bit easier we could just be typing using Spell::Spell;

It should work since C++11, which is kinda nice... If we wanna use the spell constructor, as we're doing here atm! :)